docs/releases/versions/version-v11.0.0.md

```diff
@@ -0,0 +1,122 @@
+# InvenioRDM v11.0
+
+_2023-01-05_
+
+_Short-term support (STS) release_
+
+We're happy to announce the release of InvenioRDM v11.0. The release is a short-term support release which is maintained until v11.0.
+
+## Try it
+
+- [Demo site](https://inveniordm.web.cern.ch)
+
+- [Installation instructions](https://inveniordm.docs.cern.ch/install/)
+
+## What's new?
+
+TODO
+
+## Changes
+
+### Breaking changes
+
+- TODO
+
+### Minor changes
+
+#### Docker images and helm charts
+
+TODO
+
+#### Deposit form template
+
+The deposit form template can now be overriden by editing the configuration variable `APP_RDM_DEPOSIT_FORM_TEMPLATE`.
+
+#### CLI commands
+
+##### Rebuild all search indices
+
+A new command `rebuild-all-indices` was added to `invenio rdm` command. It will rebuild the index of every service that is registered in the service registry.
+
+##### Confirm user on creation
+
+Command `invenio users create` has a new flag `--confirm`, or `-c` in short, that automatically confirms an user when created through the cli.
+
+#### Files integrity
+
+##### Checksum
+
+A task was added to automatically check whether files are corrupted, meaning that a file's checksum differs from the its original checksum.
+The celery task can be configured by editing the configuration `CELERY_BEAT_SCHEDULE['file-checks']`.
+
+##### Reports
+
+By default, everyday at 07:00 UTC a file integrity report is sent, by e-mail, in case a file is found to be corrupted.
+The celery task can be configured by editing the configuration `CELERY_BEAT_SCHEDULE['file-integrity-report']`.
+The e-mail sender and recipient can be modified by editing the following configurations:
+
+- sender: `MAIL_DEFAULT_SENDER`
+- recipient: `ADMIN_EMAIL_RECIPIENT`
```

Comment:
This will not be picked up by the app. Will be renamed to `APP_RDM_ADMIN_EMAIL_RECIPIENT` or similar at https://github.com/inveniosoftware/invenio-app-rdm/issues/2002